README.md

```diff
@@ -1,5 +1,23 @@
+<h1 align="center">
```

Comment:
In my case, Nuget didn't support `<h1>` tag in it's markdown language renderer so I was forced to remove it. 
Additionally, layout of Sinch is left aligned so maybe we can follow same guideline here?